docs/graphql/core/migrations/manage-metadata.rst

```diff
@@ -157,6 +157,41 @@ when a new column has been added to a table via an external tool such as ``psql`
    Reloading may result in inconsistent metadata status. You may need to resolve
    all inconsistent objects manually or delete them. After that, you need to reload
    metadata again.
+   
+   
+ .. _reset_metadata_manual:
+
+Reseting Hasura metadata
+-------------------------
+
+Clearing GraphQL Engine's metadata is an irreversible process, so use this functionality carefully.
+
+.. rst-class:: api_tabs
+.. tabs::
+
+  .. tab:: CLI
```

Comment:
Please make the tab in the same order as in the other examples.

Reply:
I'm not sure what you meant. The order's CLI, console, API, as in the other examples.